data_steward/tools/create_tier.py

```diff
@@ -0,0 +1,130 @@
+# Python imports
+import argparse
+import logging
+import re
+
+# Third party imports
+
+# Project imports
+
```

Comment:
Use the pipeline logging setup utility at `data_steward/utils/pipeline_logging.py`.

See usage info in `configure(level=logging.INFO, add_console_handler=False)`.

Just add it the configure call to your main function.

Reply:
added!